client-side/C#/StreetSmartArcGISPro/StreetSmartArcGISPro/AddIns/DockPanes/StreetSmart.cs

```diff
@@ -1449,14 +1452,9 @@ private async void OnImageChange(object sender, EventArgs args)
                 _crossCheck = null;
               }
             }
-
```

Comment:
Is this a safe change? I think this is related to the need to update overlay GeoJSON in case location has changed.

Reply:
fixed in a different way